templates/views/views-view-grid.tpl.php

```diff
@@ -0,0 +1,14 @@
+
+<?php
+
+/**
+ * @file
+ * Default simple view template to display a rows in a grid.
+ *
+ * - $rows contains a nested array of rows. Each row contains an array of
+ *   columns.
+ *
+ * @ingroup views_templates
+ */
+?>
+<?php require(drupal_get_path('theme', 'europa') .'/templates/grid/grid.tpl.php'); ?>
```

Comment:
I think that we should override the 'views-view-grid' theme definition properly. Meaning that we should use a preprocess for it and build a render array using the europa grid in it.